Corgibytes.Freshli.Cli.Test/Formatters/OutputFormatterTest.cs

```diff
@@ -11,99 +11,101 @@
 using YamlDotNet.Serialization;
 
 
-namespace Corgibytes.Freshli.Cli.Test.Formatters
+namespace Corgibytes.Freshli.Cli.Test.Formatters;
+public class OutputFormatterTest : FreshliTest
 {
-    public class OutputFormatterTest : FreshliTest
-    {
-        private static JsonOutputFormatter JsonFormatter { get; }
-        private static YamlOutputFormatter YamlFormatter { get; }
-        private static CsvOutputFormatter CsvFormatter { get; }
+    private static JsonOutputFormatter JsonFormatter { get; }
+    private static YamlOutputFormatter YamlFormatter { get; }
+    private static CsvOutputFormatter CsvFormatter { get; }
+
+    private static readonly MetricsResult s_metricsResultTestData;
 
-        private static readonly MetricsResult s_metricsResultTestData;
+    private static readonly IList<MetricsResult> s_metricsResultListTestData;
 
-        private static readonly IList<MetricsResult> s_metricsResultListTestData;
+    public OutputFormatterTest(ITestOutputHelper output) : base(output) { }
 
-        public OutputFormatterTest(ITestOutputHelper output) : base(output) { }
+    # pragma warning disable S3963
```

Comment:
It was disabled because it wanted all the static fields to be initialized inline, and as you can see from constructor, that is neither practical nor reasonably readable/maintainable.